lib/components/Pagination.js

```diff
@@ -2,42 +2,30 @@ import React from "react";
 import Dropdown from "./Dropdown";
 import classnames from "classnames";
 
-const LeftSVG = () => (
-  <svg className="w-5 h-5" viewBox="0 0 20 20" fill="currentColor">
-    <path
-      fillRule="evenodd"
-      d="M12.707 5.293a1 1 0 010 1.414L9.414 10l3.293 3.293a1 1 0 01-1.414 1.414l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 0z"
-      clipRule="evenodd"
-    />
-  </svg>
-);
-
-const RightSVG = () => (
-  <svg className="w-5 h-5" viewBox="0 0 20 20" fill="currentColor">
-    <path
-      fillRule="evenodd"
-      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
-      clipRule="evenodd"
-    />
-  </svg>
-);
-
-const Pagination = ({ count, pageNo, pageSize, navigate, className }) => {
-  let start = 0,
-    end = 0;
+const Pagination = ({ count = 0, pageNo, pageSize, navigate, className = "" }) => {
+  let start = 0, end = 0;
   if (count) {
     start = pageNo > 1 ? (pageNo - 1) * pageSize + 1 : 1;
     end = count < pageNo * pageSize ? count : pageNo * pageSize;
   }
-  let lastPage = false;
-  if (count < pageNo * pageSize && count > (pageNo - 1) * pageSize) {
-    lastPage = true;
-  }
+  const isLastPage = pageNo * pageSize === count;
 
   const jumpToLast = () => {
-    const lastPageIndex = parseInt(count / pageSize) + 1;
+    const lastPageIndex = Math.floor(count / pageSize);
```

Comment:
Why were these logics updated?

Reply:
@goutham-subramanyam 

`parseInt` is for parsing a string into integer. 

I didn't understand the point of having a `+ 1`. Isn't it a bug?

Eg: Let's say total count = 10. And pageSize = 5. Then the last page index = 2. Not `2+1=3`.

Do let me know if there is some other use case for `+ 1`.

Reply:
@yedhink _a Lets say we are in page 1. The total count = 9. pageSize = 5. We are calling the jumpToLast function.
(count/pageSize) provides 9/5 = 1 which is still in page 1. Hence we are providing +1.